src/debugProtocol/responses/responseCreationHelpers.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { SmartBuffer } from 'smart-buffer';
```

Comment:
Are there any additional helpers you'd like to add in this PR, or will that come in future PRs?

Reply:
I think they will come in the future as we add more tests.